packages/transformers/js/core/src/lib.rs

```diff
@@ -85,9 +72,9 @@ struct Config {
 }
 
 #[derive(Serialize, Debug, Deserialize, Default)]
-struct TransformResult<'a> {
+pub struct TransformResult {
   #[serde(with = "serde_bytes")]
-  code: &'a [u8],
+  code: Vec<u8>,
```

Comment:
Need to investigate whether this being a Vec has worse performance, and if yes, how to still use a slice

Reply:
@devongovett What did you see while profiling that motivated the change from string to `&'a [u8]`?